### DISCUSSION TITLE
ImageSharp Issue with Linux!

### DISCUSSION THREAD

**Discussion by NotThisJin** [SixLabors/ImageSharp#2662] [Q&A] [answered]
Im using ImageSharp for my discord bot and it works on windows of course but when i try to use on Linux it just doesnt do anything. I saw another post saying that it only works on .net core runtime so does that mean that i cant use .Net 6.0?

**Comment by tocsoft** [comment]
`.Net 6.0` is the `.net core runtime` they are the same thing microsoft rebranded around the .net 5.0 release. Ultimately ImageSharp workd really well on linux.

By the way you have failed to provided any sort of detail on the actual issue you are having... if your hitting errors or not. What code your running etc. I would have to guess that its something else in your code that is broken on linux and nothing to do with image sharp.

I hope you have validated your code works when skipping the imagesharp related code paths? and you have confirmed in your logs that any error are originating from inside imagesharp, please provide the errors/stack traces.

Common gotchas for linux would be path/case sensitivity differences. If your using ImageSharp.Drawing to draw text assuming that certain fonts are installed on the host and installed in locations we can detect.

**Reply by NotThisJin** [reply, marked as answer]
Sorry for the long reply as i didnt see this before, i actually just figured out why it wasnt working. I was looking for internal system fonts in my linux server and it doesnt have any so i had to path it to my folder with an added Font. Just replying now for any future people that couldnt solve this issue.
